cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -229,3 +274,13 @@ func (a *API) writeJSON(w http.ResponseWriter, r *http.Request, code int, object
 		a.log(r).Warnf("Failed to write response: %s", err)
 	}
 }
+
+func (a *API) updateInternalState(ctx context.Context, logger logrus.FieldLogger) {
+	numberOfServers, numberOfClients, err := a.db.CountEntries(ctx)
+	if err != nil {
+		logger.WithError(err).Errorf("failed to get number of clients and servers")
+		a.error = err.Error()
```

Comment:
What do you think about setting `a.error` to `""` if `a.db.CountEntries(ctx) returns no error?

Reply:
Good idea. Done